frontend/src/apis/create-api.ts

```diff
@@ -0,0 +1,32 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { UserProfile } from 'models';
+
+enum ApiTag {
+  PROFILE = 'Profile',
+}
+export interface Post {
```

Comment:
Why is this named `Post`? Isn't `Post` a specific model in the case of this app?